ADS Fetcher (Test) fails

Latest dev version https://github.com/JabRef/jabref/commit/955b94d7221e5ee3c102731f885eb395e1c2ad64

From what I see, the entry returned from the fetcher contains the field    `eid = {#10}`
```

java.lang.AssertionError: expected:<Optional[@article{2012LRR....15...10F,
  archiveprefix = {arXiv},
  author = {Famaey, B. and McGaugh, S. S.},
  doi = {10.12942/lrr-2012-10},
  eprint = {1112.3960},
  journal = {Living Reviews in Relativity},
  keywords = {astronomical observations, Newtonian limit, equations of motion, extragalactic astronomy, cosmology, theories of gravity, fundamental physics, astrophysics},
  month = {#sep#},
  title = {Modified Newtonian Dynamics (MOND): Observational Phenomenology and Relativistic Extensions},
  volume = {15},
  year = {2012}
}]> but was:<Optional[@article{2012LRR....15...10F,
  archiveprefix = {arXiv},
  author = {Famaey, B. and McGaugh, S. S.},
  doi = {10.12942/lrr-2012-10},
  eid = {#10},
  eprint = {1112.3960},
  journal = {Living Reviews in Relativity},
  keywords = {astronomical observations, Newtonian limit, equations of motion, extragalactic astronomy, cosmology, theories of gravity, fundamental physics, astrophysics},
  month = {#sep#},
  pages = {#10},
  title = {Modified Newtonian Dynamics (MOND): Observational Phenomenology and Relativistic Extensions},
  volume = {15},
  year = {2012}
}]>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at net.sf.jabref.logic.importer.fetcher.AstrophysicsDataSystemTest.testPerformSearchByFamaeyMcGaughEntry(AstrophysicsDataSystemTest.java:161)
```

--- a/src/test/java/net/sf/jabref/logic/importer/fetcher/AstrophysicsDataSystemTest.java
+++ b/src/test/java/net/sf/jabref/logic/importer/fetcher/AstrophysicsDataSystemTest.java
@@ -74,7 +74,9 @@ public class AstrophysicsDataSystemTest {
         famaeyMcGaughEntry.setField("month", "#sep#");
         famaeyMcGaughEntry.setField("archiveprefix", "arXiv");
         famaeyMcGaughEntry.setField("doi", "10.12942/lrr-2012-10");
+        famaeyMcGaughEntry.setField("eid", "#10");
         famaeyMcGaughEntry.setField("eprint", "1112.3960");
+        famaeyMcGaughEntry.setField("pages", "#10");
         famaeyMcGaughEntry.setField("keywords", "astronomical observations, Newtonian limit, equations of motion, extragalactic astronomy, cosmology, theories of gravity, fundamental physics, astrophysics");
 
         sunWelchEntry = new BibEntry();

